libtsuba/include/katana/EntityTypeManager.h

```diff
@@ -597,6 +597,19 @@ class KATANA_EXPORT EntityTypeManager {
   /// std::string PrintTypes() IS A TESTING ONLY FUNCTION, DO NOT EXPOSE THIS TO THE USER
   std::string PrintEntityTypes() const;
 
+  size_t ApproxByteCount() const {
```

Comment:
Is measuring this (memory usage of EntityTypeManager) important? If yes, then we should include the size of dynamic allocated things like strings and dynamic bitsets in the maps.

Reply:
Because I was measuring the space consumed by properties, I figured I would measure types also.  But the encapsulation is poor here, so I'm thinking it is not worthwhile.

Reply:
I agree. This would only matter if we really had a lot of Entity types with really long names. On the other hand, it does make sense to account for Entity Type ID Arrays in `PropertyGraph`. I assume you're doing that with `graph-info`?

Reply:
I do not account for EntityTypeID arrays in graph-info.  You can add that feature!